### PR TITLE
refactor: use updated identify payload instead

### PR DESCRIPTION
**Pull request by ayntee**

### CODE REVIEW

File: src/bot.ts [hunk 1]
```diff
@@ -29,18 +30,6 @@ export const identifyPayload: IdentifyPayload = {
   shard: [0, 0],
 };
 
-export interface IdentifyPayload {
```
Comment (Skillz4Killz) [inline src/bot.ts:32]: Bring this back and mark it deprecated to make this pr non-breaking

Comment (ayntee) [reply src/bot.ts:32]: @Skillz4Killz the types are not supposed to be private, hence not supposed to be used by the users, but internally.

Comment (Skillz4Killz) [reply src/bot.ts:32]: That doesnt mean its not part of the public api. Realistically, you correct. Technically, this is still breaking.

Comment (ayntee) [reply src/bot.ts:32]: > That doesnt mean its not part of the public api. Realistically, you correct. Technically, this is still breaking.

I don't really mind, but just putting that out there...

File: src/bot.ts [hunk 2]
```diff
@@ -29,18 +30,6 @@ export const identifyPayload: IdentifyPayload = {
   shard: [0, 0],
 };
```
Comment (ayntee) [inline src/bot.ts:32]:
```suggestion
/** deprecated Use "DiscordIdentify" instead */
export interface IdentifyPayload {
  token: string;
  compress: boolean;
  properties: {
    $os: string;
    $browser: string;
    $device: string;
  };
  intents: number;
  shard: [number, number];
}
```

File: src/bot.ts [hunk 3]
```diff
@@ -28,7 +29,7 @@ export const identifyPayload: IdentifyPayload = {
   intents: 0,
   shard: [0, 0],
 };
```
Comment (ayntee) [inline src/bot.ts:31]:
```suggestion
};

```

File: src/bot.ts [hunk 4]
```diff
@@ -29,6 +30,7 @@ export const identifyPayload: IdentifyPayload = {
...
 
+/** deprecated Use "DiscordIdentify" instead */
```
Comment (ayntee) [inline src/bot.ts:33]:
```suggestion
/** @deprecated Use "DiscordIdentify" instead */
```

File: src/bot.ts [hunk 5]
```diff
@@ -40,7 +42,6 @@ export interface IdentifyPayload {
   intents: number;
   shard: [number, number];
 }
```
Comment (ayntee) [inline src/bot.ts:44]:
```suggestion
}

```

**Pull request merged**